airsonic-main/src/main/java/org/airsonic/player/service/JWTSecurityService.java

```diff
@@ -80,4 +86,26 @@ public static DecodedJWT verify(String jwtKey, String token) {
     public DecodedJWT verify(String credentials) {
         return verify(settingsService.getJWTKey(), credentials);
     }
+
+
+    /**
+     * Create a unexpired token for sonos link
```

Comment:
Do you mean to say:
-  "an unexpiring token", as in "it will never expire"
-  "an unexpired token", as in "it have not expired yet"
?

Reply:
:-( I'm not very good in english!

an unexpiring token", as in "it will never expire"

I copy paste the comment.